defold-spine/commonsrc/spine_loader.cpp

```diff
@@ -374,7 +374,7 @@ namespace dmSpine
         {
             loader->error1 = strdup(skeleton_json->error ? skeleton_json->error : "unknown error");
             spSkeletonJson_dispose(skeleton_json);
-            dmLogError("Failed to read spine skeleton for %s: %s", path, skeleton_json->error);
+            dmLogError("Failed to read spine skeleton for %s: %s", path, loader->error1);
```

Comment:
this var freed a line above, and can't be used. `loader->error1` should be used instead